avm/ptn/dev-ops/cicd-agents-and-runners/tests/e2e/private-devops-existing-vnet-aci/main.test.bicep

```
@@ -0,0 +1,87 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults for Azure DevOps self-hosted agents using Private networking in an existing vnet.'
+metadata description = 'This instance deploys the module with the minimum set of required parameters Azure DevOps self-hosted agents using Private networking in Azure Container Instances in an existing vnet.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-azuredevops-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Required. The personal access token for the Azure DevOps organization.')
```

```suggestion
@description('Optional. The personal access token for the Azure DevOps organization.')
```

Given that you mentioned the value can be a dummy value (and due to the default it is), let's reflect this in the description. In this and the other tests

---

Resolved